templates/symfony-6/.github/workflows/build_release.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    tags:
+      - "*.*.*"
+
+name: Create Github Release
+
+permissions:
+  contents: write
+
+env:
+  COMPOSE_USER: runner
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    env:
+      COMPOSER_ALLOW_SUPERUSER: 1
+      APP_ENV: prod
```

Comment:
Are these actually needed/used, i.e. do the have any effect? We set `APP_ENV` when running `docker compose run` (in lines 27–28).